packages/typespec-ts/test/integration/generated/type/union/src/index.d.ts

```diff
@@ -253,9 +273,7 @@ export declare interface StringAndArrayGet {
 
 export declare interface StringAndArrayGet200Response extends HttpResponse {
     status: "200";
-    body: {
-        prop: StringAndArrayCasesOutput;
-    };
+    body: StringAndArrayCasesOutput;
```

Comment:
is this from cadl ranch change?

Reply:
will use this pr to track the RLC generation fix https://github.com/Azure/autorest.typescript/pull/2896